util/cache/single.go

```diff
@@ -25,6 +25,8 @@ type Single[K comparable, V any] struct {
 	ServeExpired bool
 }
 
+var _ Cache[int, int] = &Single[int, int]{}
```

Comment:
can write this without allocating (or looking like it is) by making the RHS be `(*Single[int, int])(nil)` ?